mkpkg

```diff
@@ -159,6 +159,9 @@ class Functions:
         return sorted(deps)
 
     def mapfiles(self, src, dst, *files, **kwargs):
+        if not files:
+            die("'FUN.mapfiles()' expects at least one file to be passed")
+
```

Comment:
This is a breaking change so it should go to v2.x.x. But I don't understand the rationale for this... What's the point exactly?